docs/qunit-options.md

```diff
@@ -36,6 +36,12 @@ Default: `false`
 
 When true, the whole task will not fail when there are individual test failures, or when no assertions for a test have run. This can be set to true when you always want other tasks in the queue to be executed.
 
+## summaryOnly
+Type: `boolean`  
+Default: `false`
```

Comment:
You should add this option to the default options (line `158 => 170`), otherwise its default value is `undefined`